checks.d/mysql.py

```diff
@@ -896,13 +896,19 @@ def _get_stats_from_innodb_status(self, db):
         try:
             with closing(db.cursor()) as cursor:
                 cursor.execute("SHOW /*!50000 ENGINE*/ INNODB STATUS")
-                innodb_status = cursor.fetchone()
-                innodb_status_text = innodb_status[2]
         except (pymysql.err.InternalError, pymysql.err.OperationalError, pymysql.err.NotSupportedError) as e:
             self.warning("Privilege error or engine unavailable accessing the INNODB status \
                          tables (must grant PROCESS): %s" % str(e))
             return {}
 
+        if cursor.rowcount < 1:
```

Comment:
I'm wondering if you get an empty output because the user we're querying with does not have enough privileges to read the DB. I believe `SHOW /*!50000 ENGINE*/ INNODB STATUS` requires the user to have `PROCESS` privileges... That doesn't necessarily change the scope of this PR but is probably worth knowing.